doc/ReleaseNotes/_ReleaseNotes.md

```diff
@@ -107,7 +107,11 @@
     > This change might break the compilation for projects that define duplicate resources in other globally accessible resource dictionaries. Adjustments to remove duplicate resources may be necessary.
 
 ### Bug fixes
+<<<<<<< HEAD
```

Comment:
Warning warning bad merge

Reply:
Thanks you! correcting don't know our I provoked this!